hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -63,6 +63,10 @@
 
   public HiveSyncTool(HiveSyncConfig cfg, HiveConf configuration, FileSystem fs) {
     super(configuration.getAllProperties(), fs);
+    //Set Hive meta store URI which enables synchronize to remote hive
+    if (cfg.hiveMetaStoreUri != null) {
```

Comment:
I see that we set a default value for this new config in DataSourceUtils( thrift://localhost:9083). So, for someone who does not set this, I assume it should be no-op even though we set to some default value. I don't have much knowledge on hive megastore and sync. So, might be a naive question.